examples/kubernetes/README.md

```diff
@@ -30,7 +30,7 @@ If you're running in Container Engine, set the `KUBECTL` environment variable
 to point to the `gcloud` command:
```

Comment:
We should change this to something like:

to point to the `kubectl` command provided by the Google Cloud SDK.
If you've already added `gcloud` to your `PATH`, you probably also have `kubectl`.